pkgs/applications/editors/emacs/elisp-packages/manual-packages/cask/package.nix

```diff
@@ -38,30 +53,23 @@ stdenv.mkDerivation (finalAttrs: {
     s
     servant
     shell-split-string
-  ]);
-
-  strictDeps = true;
-
-  buildPhase = ''
-    runHook preBuild
+  ];
 
-    emacs --batch -L . -f batch-byte-compile cask.el cask-cli.el
+  propagatedUserEnvPkgs = [
+    coreutils # cask needs readlink
```

Comment:
It looks like you choose the first method I mentioned [here](https://github.com/NixOS/nixpkgs/pull/338010/files#r1735648450).  Note than readlink is not used on darwin, `python` is used instead.  Similarly, `python` is not used on non-darwin systems.

Would you consider switching to the second method which makes things a lot easier?

Reply:
Ah, now I understood. I thought you suggested to set SRCDIR _outside_ as if it was an env var...